server/src/test/java/com/tisza/tarock/util/util/Util.java

```diff
@@ -0,0 +1,19 @@
+package com.tisza.tarock.util.util;
```

Comment:
I think you mistakenly placed it in com.tisza.tarock.util.util instead of com.tisza.tarock.util.

Reply:
Yes, I missed it.